### PR TITLE
Enhancement: Add cronos testnet to etherscan source fetcher

### PR DESCRIPTION
**Pull request by haltman-at**
Looks like Etherscan just added this network.  Adding it here.

### CODE REVIEW

File: packages/source-fetcher/lib/etherscan.ts [hunk 1]
```diff
@@ -75,6 +75,7 @@ const EtherscanFetcher: FetcherConstructor = class EtherscanFetcher
       "moonbase-alpha": "api-moonbase.moonscan.io",
       "hoo": "api.hooscan.com",
       "cronos": "api.cronoscan.com",
+      "testnet-cronos": "api-testnet.cronoscan.com",
```
Comment (cds-amal) [inline packages/source-fetcher/lib/etherscan.ts:78]: api-testnet.cronoscan.com
```
invalid resource end point
Test#1: [api?module=account&action=balance&address=0xddbd2b932c763ba5b1b7ae3b362eac3e8d40121a&tag=latest&apikey=YourApiKeyToken](https://api-testnet.cronoscan.com/api?module=account&action=balance&address=0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae&tag=latest&apikey=YourApiKeyToken)
Test#2: [api/api?module=account&action=txlist&address=0xddbd2b932c763ba5b1b7ae3b362eac3e8d40121a&startblock=0&endblock=99999999&page=1&offset=10&sort=asc&apikey=YourApiKeyToken](https://api-testnet.cronoscan.com/api?module=account&action=txlist&address=0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae&startblock=0&endblock=99999999&page=1&offset=10&sort=asc&apikey=YourApiKeyToken)
```

I tried that endpoint which asks for `YourApiKeyToken`. Is that the apiKey specified under the etherscan namespace in truffle-config?

```javascript
  etherscan: {
    apiKey: process.env.ETHERSCAN_KEY
  }
```

**Review by cds-amal — COMMENTED** [review]
Looks good @haltman-at! I have one question.

File: packages/source-fetcher/lib/etherscan.ts (hunk 1 above)
Comment (cds-amal) [reply packages/source-fetcher/lib/etherscan.ts:78]: Nice branch name! 😂

Comment (haltman-at) [reply packages/source-fetcher/lib/etherscan.ts:78]: Yes, this is the same for the etherscan fetcher as a whole.  The API key isn't required, but it reduces the rate limiting (something we account for).

Comment (cds-amal) [reply packages/source-fetcher/lib/etherscan.ts:78]: Thanks!

**Pull request merged**